src/main/java/com/group6/harmoniq/controllers/UserController.java

```diff
@@ -63,9 +63,17 @@ public String setAdminStatus(@RequestParam("spotifyId") String spotifyId, @Reque
                 user.setIsAdmin(isAdmin);
                 userRepository.save(user);
             }
-            return "redirect:/admin/users";
+            return "redirect:/userList";
         } else {
-            return "redirect:/access-denied"; // Redirect to an access-denied page
+            return "redirect:/accessDenied"; // Redirect to an access-denied page
+        }
+    }
+
+    public void incrementUserQuizCount(HttpSession session) {
```

Comment:
Since, it's not a public endpoint, can we move this to `UserService`?

Reply:
See commit: 81167ee06958d8318c5c5b5d2fe394073f5b3a37